src/Packetery/Module/Order/Controller.php

```diff
@@ -165,7 +175,7 @@ public function saveModal( WP_REST_Request $request ) {
 			'packetery_length'     => $order->getLength(),
 			'packetery_width'      => $order->getWidth(),
 			'packetery_height'     => $order->getHeight(),
-			'showWarningIcon'      => $this->orderModal->showWarningIcon( $order ),
+			'orderIsSubmittable'   => $this->orderValidator->validate( $order ) && ( $order->getFinalWeight() > 0 ),
```

Comment:
Podmínku `$this->orderValidator->validate( $order ) && ( $order->getFinalWeight() > 0 )` máš na 2 místech.  Pokud bude někdy potřeba změnit podmínky za kterých je možné zásilku odeslat,  musíš to změnit na všech místech, kde to používáš.  Odpovědnost za to, zda lze zásilku odeslat by měl mít jen validátor.